plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -182,6 +182,7 @@
 public final class IcebergUtil
 {
     public static final String TRINO_TABLE_METADATA_INFO_VALID_FOR = "trino_table_metadata_info_valid_for";
+    public static final String TRINO_NOT_CACHEABLE_TABLE_COMMENT = "trino_too_long_table_comments";
```

Comment:
The value `trino_too_long_table_comments` is obsolete. 
How about `TRINO_TABLE_COMMENT_IS_CACHED` with `trino_table_comment_is_cached`?

Reply:
That won't work, this property needs to be set to distinguish situation when comment was not cached because it is too long with situation when comment is just not there. Having it set when comment is cached is not beneficial because it would be the same as `TABLE_COMMENT` which is already there. 

In the current version i forgot to change the actual value of the property but I still believe this is the right approach and we just maybe need a better naming for it

Reply:
I will rename to `TRINO_TABLE_COMMENT_CACHE_PREVENTED` and this time I will change the actual value too